src/datastore/postgres-store.ts

```diff
@@ -4230,6 +4231,28 @@ export class PgDataStore
     });
   }
 
+  async getTxsCount({ from, to }: { from: number; to: number }) {
+    let resultQuery: QueryResult<TransactionsCount>;
+    return this.queryTx(async client => {
+      const maxHeight = await this.getMaxBlockHeight(client, { includeUnanchored: true });
+      resultQuery = await client.query<TransactionsCount>(
+        `
+          SELECT DATE_TRUNC('day', to_timestamp(parent_burn_block_time)) as "day", count(*)::int as count
```

Comment:
I think we'll need to run some performance tests on this query given that `COUNT(*)` is a very expensive operation and the `txs` table is the largest in the DB (currently at ~2.5 million rows IIRC). Maybe keeping a materialized view with these counts would make sense.

Reply:
I concure, now how do we go about doing that